cmd/abci-cli/abci-cli.go

```diff
@@ -271,6 +274,16 @@ var dummyCmd = &cobra.Command{
 	},
 }
 
+var testCmd = &cobra.Command{
+	Use:   "test",
+	Short: "Run integration tests",
+	Long:  "",
```

Comment:
can we get a long description here? Even if it's the same as Short, they show up differently

Reply:
None of the others have it. Feel free to add them tho :)